tools/build-tools/src/bumpVersion/bumpVersionCli.ts

```diff
@@ -65,8 +65,8 @@ function parseNameVersion(arg: string | undefined) {
 
     let version: VersionChangeType | undefined;
     if (v !== undefined) {
-        if (v === "minor" || v === "patch") {
-            version = v;
+        if (["major", "minor", "patch"].includes(v)) {
```

Comment:
can this be a constant array of type VersionBumpType to remove the below cast

Reply:
actually ts still wants a cast somewhere because `v` is still a string and not directly comparable to `VersionBumpType`... it may be possible with some more refactoring of the `VersionBumpType` definition which I didn't try; will revisit when adding full support for the major type in the bump tooling